src/algorithm/REDUCE_SUM-Cuda.cpp

```diff
@@ -218,6 +218,49 @@ void REDUCE_SUM::runCudaVariantRAJA(VariantID vid)
 
 }
 
+template < size_t block_size, typename MappingHelper >
+void REDUCE_SUM::runCudaVariantRAJANewReduce(VariantID vid)
+{
+  using exec_policy = std::conditional_t<MappingHelper::direct,
+      RAJA::cuda_exec<block_size, true /*async*/>,
+      RAJA::cuda_exec_occ_calc<block_size, true /*async*/>>;
```

Comment:
We should probably switch these to the with_reduce policy.

Reply:
Based on the discussion at the meeting this morning, I thought you wanted to hold off on changing RAJA policies for reduction kernels for now. I didn't change any policies that were already there.

Reply:
We can do it later, I forgot about this particular wrinkle in the reduction tunings.